### PR TITLE
Dependencies: remove behance preset and add old jasmine peer dep.

### PR DESCRIPTION
**Pull request by mikesherov**

### CODE REVIEW

File: lib/localTasks.js [hunk 1]
```diff
@@ -1,5 +1,18 @@
 var path = require('path');
 
 exports.load = function(grunt, plugin) {
-  grunt.loadTasks(path.join(__dirname, '..', 'node_modules', plugin, 'tasks'));
+  var nestedTaskPath = path.join(__dirname, '..', 'node_modules', plugin, 'tasks');
+  var dedupedTaskPath = path.join(__dirname, '..', '..', '..', 'node_modules', plugin, 'tasks');
```
Comment (Aintaer) [inline lib/localTasks.js]: oh my god how are you soo ballsy

Comment (Aintaer) [inline lib/localTasks.js]: How do you even know that `../../../node_modules` exists???

Comment (mikesherov) [inline lib/localTasks.js]: It'll explode if it doesn't. Which is fine.

**Comment by Aintaer** [comment]
:+1:

**Comment by mrjoelkemp** [comment]
:+1:

**Pull request merged**